analysis_driver/dataset.py

```diff
@@ -478,6 +478,12 @@ def _amount_data(self):
             ]
         )
 
+    @property
+    def pc_Q30(self):
```

Comment:
PEP8, plz (lower case)

Reply:
Will do, thanks! Aware of the second part of this, will be added in a separate commit.